df.history.xml

```diff
@@ -311,14 +311,29 @@
                 <pointer>
                     <int32_t name='histfig_id' ref-target='historical_figure'/>
                     <int32_t/>
-                    <stl-vector type-name='int32_t'/>
-                    <stl-vector type-name='int32_t'/>
-                    <stl-vector type-name='int32_t'/>
+                    <stl-vector name="attitude">
+                        <enum base-type='int32_t' type-name='reputation_type' comment="Probably ordered"/>
```

Comment:
base-type technically isn't necessary because of https://github.com/DFHack/df-structures/blob/f0c609211fbfd354b576cae75e9e64f8c6c80b06/df.history.xml#L39

Reply:
I didn't think it should be, but since that was the format used in 'reputation' just above I decided to play it safe and use the same pattern.